### PR TITLE
docs: added Domoticz as application using the MQTT Discover function.

### PR DESCRIPTION
**Pull request by waltervl**
See also https://www.domoticz.com/wiki/ZwaveJS2MQTT

### CODE REVIEW

**Comment by robertsLando** [comment]
@waltervl Please add it as a new entry in the list

**Comment by waltervl** [comment]
Thank you for reviewing. Modified as requested.

File: docs/README.md [hunk 1]
```diff
@@ -45,4 +45,5 @@ Fully configurable Z-Wave to MQTT **Gateway** and **Control Panel**.
 - **Access Store Files in the UI**: Access the files are stored in the persistent `store` folder directly from the UI
 - **Network Graph**: Provides a beautiful map showing how nodes are communicating with the controller
 - **Supports the Official Home Assistant Integration**: Can act as the backend driver for the official Home Assistant integration, using the same driver and socket server as the official addon
-- **Supports Home Assistant Discovery via MQTT**: In lieu of the official integation, can be used to expose Z-Wave devices to Home Assistant via MQTT discovery
+- **Supports Home Assistant Discovery via MQTT**: In lieu of the official integation, can be used to expose Z-Wave devices to Home Assistant via MQTT discovery. 
+- **Supported by Domoticz (beta 2021.1) using MQTT Autodiscovery.
```
Comment (robertsLando) [inline docs/README.md:49]:
```suggestion
- **Supported by Domoticz** (beta 2021.1) using MQTT Autodiscovery. 
```

**Review by robertsLando — CHANGES_REQUESTED** [review]

**Comment by waltervl** [comment]
It should be resolved now or did I miss something in the github review process?

**Review by robertsLando — APPROVED** [review]

**Pull request merged**